Makefile

```diff
@@ -105,7 +105,7 @@ export LEG9
 # What alpine image does the vlogger image use
 VLOGGER_BASE_IMG?=alpine
 # What version of alpine does the vlogger image use
-VLOGGER_ALPINE_VERSION?=3.19
+VLOGGER_ALPINE_VERSION?=3.18
```

Comment:
What was this about?

Reply:
If you see the error log of the previous commit, e2e tests cannot pull "alpine:3.19" since 3.19 doesn't support AMD. I will switch it back to 3.19 when alpine fixed that error.